eStore/Core/UI/LightTheme.swift

```diff
@@ -16,5 +16,8 @@ final class LightTheme: Theme {
     let foregroundColor = Color.white
     let borderColor = Color.silver
     let textColor = Color.black
-    let tintColor = Color.azraqBlue
+    var tintColorType: TintColorType = .azraqBlue
```

Comment:
I would suggest making it an `init argument`... You can always recreate a struct if you need to change this property, mutability is kind of an inconvenient thing to keep track of later on... 
Personally i would make an `init` and get rid of `var`